cloud_governance/tag_user/iam_user_tags.py

```diff
@@ -0,0 +1,80 @@
+import boto3
+
+from cloud_governance.common.aws.iam.iam_operations import IAMOperations
+from cloud_governance.common.elasticsearch.elasticsearch_operations import ElasticSearchOperations
+from cloud_governance.common.logger.init_logger import logger
+
+
+class ValidateIAMUserTags:
+
+    def __init__(self, es_host: str, es_port: str, es_index: str):
+        self.__es_host = es_host
+        self.__es_port = es_port
+        self.__es_index = es_index
+        if self.__es_host and self.__es_port:
+            self.__elastic_search_operations = ElasticSearchOperations(es_host=self.__es_host, es_port=self.__es_port)
+        self.iam_client = boto3.client('iam')
+        self.iam_operations = IAMOperations()
+
+    def check_trail_spaces(self, tags: list, ):
+        """
+        Check trail spaces
+        @param tags:
+        @return:
+        """
+        new_tags = []
+        if tags:
+            for tag in tags:
+                if tag.get('Key').startswith(' ') or tag.get('Key').endswith(' '):
+                    new_tags.append(tag.get('Key').strip())
+        return new_tags
+
+    def upload_trailing_user_tags(self):
+        """
+        This method gets the trailing spaces tags of each user
+        @return:
+        """
+        users = self.iam_operations.get_users()
+        output_data = []
+        for user in users:
+            if '-' not in user['UserName']:
```

Comment:
For what we need this check ?

Reply:
We know User have without **-.**

Reply:
You mean that the user value doesnt include -. right 
What will be if we have a user with this char ?

Reply:
We create a user without -, With - means it's not created by us manually. except for our cloud-governance

Reply:
DO we not have other way to find it ?

Reply:
we do it same for IAM user tags update.